tests/conftest.py

```diff
@@ -40,6 +48,12 @@ def pytest_addoption(parser):
         default=True,
         help="Whether to run container in detached mode",
     )
+    parser.addoption(
+        "--ignore-filters",
+        action="store_true",
+        default=False,
+        help="Don't filter tests by marks.",
```

Comment:
nit: by levels?

Reply:
I think this is correct, it will also apply to other marks